docs/ZXing_barcode.md

```diff
@@ -0,0 +1,73 @@
+
+To test the barcode scanner specifically, head to the Upload Documents section. At the drop down form under "Select Document Type", select "Driver's License". You may optionally provide an expiration date for the document and also a description. Finally under the "File to upload" section, select the scanned image of the Driver's License PDF417 barcode from your computer. Click on the "Upload file" button and wait until you receive a short confirmation message below confirming your file has been uploaded.
+
+To see the results on the Solid Client Server, navigate below to the Search Document section. There you will find a form drop down titled "Select Document Type". Select the "Driver's License" option and once selected, click on the "Get Document" button. After some loading time, a url should appear within this section of the page. Click on that link and a new tab will open which will present you with an interface for the Solid Pod. Here you will find icons that will navigate you to the scanned barcode image as well as the .ttl file containing the scanned data from the Driver's License barcode.
+
+**Entry Point to the Code**
+
+For developers new to a code base, often being pointed to where user interaction first interfaces with the code base is a good place to start. In the case of the barcode scanner functionality, this can be found within the UploadDocumentForm.jsx component within the src/components directory:
+
+```
+src/components/Form/UploadDocumentForm.jsx
+```
+
+At the time of this writing, the line specific to uploading files can be found on line 67 of this file, where the entry point of what happens when the user clicks on the "Upload file", as this triggers the handleFormSubmission function, defined on line 48. This function in turn calls the makeHandleFormSubmission function found within the utils directory, where the majority of the barcode scanner logic lives:
+
+```
+src/utils/frontend/FormSubmissionHelper.js
+```
+
+This function handles a good deal of logic for submitting a wide variety of forms to the Solid Server, including parsing the fileObject passed based off the event, and most importantly uploading the document, specifically by calling the uploadDocument() function defined on line 47. Going to the definition of this function leads us to:
+
+```
+src/utils/network/session-core.js
+```
+
+Defined on line 162, uploadDocument() contains a great deal of logic specific to solid and we start to see a good deal of functions coming from the [@inrupt/solid-client libraries](https://docs.inrupt.com/developer-tools/javascript/client-libraries/reference/solid-client/). Of particular interest to the subject of this document is the createResourceTtlFile() function on line 191, which is defined within:
```

Comment:
line is 119 or 131

Reply:
Think we can use just the function name since specific line numbers where the function is called in a file is subject to change during refactors/optimizations.